schemata/organization.json

```diff
@@ -34,7 +34,7 @@
         "type": "object",
         "properties": {
           "id": {
-            "type": "integer",
+            "type": "bigint",
```

Comment:
I'm fairly certain this isn't a valid type in JSON Schema.  It should be left as "integer".  [Related StackOverflow answer.](http://stackoverflow.com/questions/34526024/mysql-bigint-vs-json-schema-integer#34575827)